tests/cli/test_get.py

```diff
@@ -48,6 +48,7 @@ def test_invalid_arguments(self):
     # MARK: All Data
 
     @patch("scribe_data.cli.get.query_data")
+    @patch("builtins.input", lambda _: "N")  # don't use dump
```

Comment:
@axif0: Note that this is how we can be passing input arguments to tests, with in this case `"N"` being passed to say that we want to query.